mix.exs

```diff
@@ -0,0 +1,22 @@
+defmodule OpenTelemetry.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :opentelemetry_api,
+      version: "0.1.0",
```

Comment:
This is going to be an issue. Either this should be changed to query `git` for a version like rebar3 does for the `.app.src` or we should have a `VERSION` file that both read from.

Reply:
AFAIK there is nothing built into mix to do that, so a `VERSION` file sounds like the best choice to me.

Reply:
Right, it would have to be functions in mix.exs that do the `git` commands. But then would also have to have a special pre-publish thing that replaces it with the package version, ugh.

So yea, file sounds like the best choice.

Reply:
@zachdaniel I've created a VERSION file in this PR https://github.com/open-telemetry/opentelemetry-erlang-api/pull/8 so this should change to read in the version from the file.